Apps/W1/ShopifyConnector/app/Order handeling/Tables/ShpfyOrdersToImport.Table.al

```diff
@@ -0,0 +1,207 @@
+    fields
+    {
+        field(1; "Entry No."; Integer)
+        {
+            AutoIncrement = true;
+            Caption = 'Entry No.';
+            DataClassification = SystemMetadata;
+        }
+
+        field(2; ShopId; Integer)
+        {
+            Caption = 'Shop Id';
+            DataClassification = CustomerContent;
+        }
+
+        field(3; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = CustomerContent;
+            TableRelation = "Shpfy Shop".Code;
+        }
+
+        field(4; Id; BigInteger)
+        {
+            Caption = 'Id';
+            DataClassification = CustomerContent;
+        }
+
+        field(5; "Order No."; Text[50])
+        {
+            Caption = 'Order No.';
+            DataClassification = CustomerContent;
+        }
+
+        field(6; "Created At"; DateTime)
+        {
+            Caption = 'Created At';
+            DataClassification = CustomerContent;
+        }
+
+        field(7; "Updated At"; DateTime)
+        {
+            Caption = 'UpdatedAt';
+            DataClassification = CustomerContent;
+        }
+
+        field(8; Test; Boolean)
+        {
+            Caption = 'Test';
+            DataClassification = CustomerContent;
+        }
+
+        field(9; Confirmed; Boolean)
+        {
+            Caption = 'Confirmed';
+            DataClassification = CustomerContent;
+        }
+
+        field(10; "Fully Paid"; Boolean)
+        {
+            Caption = 'Fully Paid';
+            DataClassification = CustomerContent;
+        }
+
+        field(11; Unpaid; Boolean)
+        {
+            Caption = 'Unpaid';
+            DataClassification = CustomerContent;
+        }
+
+        field(12; "Risk Level"; enum "Shpfy Risk Level")
+        {
+            Caption = 'Risk Level';
+            DataClassification = CustomerContent;
+        }
+        field(13; "Financial Status"; enum "Shpfy Financial Status")
+        {
+            Caption = 'Financial Status';
+            DataClassification = CustomerContent;
+        }
+
+        field(14; "Fulfillment Status"; enum "Shpfy Order Fulfill. Status")
+        {
+            Caption = 'Fulfillment Status';
+            DataClassification = CustomerContent;
+        }
+
+        field(15; "Order Amount"; Decimal)
+        {
+            Caption = 'Order Amount';
+            DataClassification = CustomerContent;
+        }
+
+        field(16; "Currency Code"; Code[10])
+        {
+            Caption = 'Currency Code';
+            DataClassification = CustomerContent;
+        }
+
+        field(17; "Number of Items"; Integer)
+        {
+            Caption = 'Number of Items';
+            DataClassification = CustomerContent;
+        }
+
+        field(18; "Number of Order Lines"; Integer)
+        {
+            Caption = 'Number of Order Lines';
+            DataClassification = CustomerContent;
+        }
+
+        field(19; Tags; Text[1024])
```

Comment:
Question: will it make sense to increase field length?

Reply:
Increased to 2048.